.github/ISSUE_TEMPLATE/7_cherry_pick.md

```diff
@@ -0,0 +1,24 @@
+---
+name: Request a cherry-pick.
+about: As a contributor, you would like to request that a feature be cherry-picked into a release.
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+commit(s) to merge: (Enter the shortened commit hash of the commit that has been merged to master/main. <fabfeed>)
+
+merge instructions: clean merge
+
+What is the issue: (Enter brief description of the issue)
```

Comment:
The issue is probably already described in the linked GitHub issue.  More relevant for a cherry-pick would be questions about the customer impact.
- Approximately who will hit this issue (ex. all Flutter devs, Windows developers, all end-customers, apps using X framework feature)?
- What is the impact (ex. visual jank on Samsung phones, app crash, cannot ship an iOS app)? Does it impact development (ex. `flutter doctor` crashes when Android Studio is installed), or the shipping production app (the app crashes on launch)
- Is there a workaround?

Reply:
I don't think this is "community", which I think of as Flutter contributor and developers, but often end-users of the app who don't know or care that the app is using Flutter.  I said "customer" because it either impacts developers, who are our customers, or end-users, who are the developer's customers.  I'm not wed to the word though, the point is "what is the impact and who does it impact". 

If this doesn't get fixed, how bad is it?  Did the developer's app get corrupted? Does the app crash on launch? Does it only crash on the latest version of Android?  Can the developer simply work around this issue, like by upgrading their IDE? etc.

Overall I think the questions a cherry-picker needs for approval are:
1. What bug does this fix?
2. What are the pros (impact, to whom)?
3. What are the cons (risk)?